Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -282,23 +282,23 @@ final class LayerTests: XCTestCase {
         let layer = GlobalMaxPool1D<Float>()
         let input = Tensor(shape: [1, 10, 1], scalars: (0..<10).map(Float.init))
         let output = layer.inferring(from: input)
-        let expected = Tensor<Float>([9])
+        let expected = Tensor<Float>([[9]])
```

Comment:
These are unrelated changes wrt to this PR, Are they absolutely needed? Because the tests passed before the change as well. But if it's still necessary I'd suggest opening a separate PR since they are layer test fixes.

Reply:
`GlobalMaxPool1D` outputs tensor which has shape [BatchSize, Channels].
In here input shape is `[1, 10, 1]` and output shape is `[1, 1]`.
I disabled broadcasting in this PR so this test doesn't pass without change.